### ISSUE TITLE
Error throwing on Unity 2017.1.2f1

### ISSUE THREAD

**Issue by eluchsinger** [Microsoft/MixedRealityToolkit-Unity#1354]
## Overview
Backwards compatibility missing.

The missing [method](https://docs.unity3d.com/2017.2/Documentation/ScriptReference/XR.WSA.HolographicSettings.IsDisplayOpaque.html) is to know if it's a HoloLens device or a VR device. The simplest fix (although with maybe unintended consequences) would be to wrap it in an `#if`. But if MR is supported after 2017.2, this could be a viable way to do it.

## Expected Behavior
There should be no errors after opening the project.

## Actual Behavior
The following error appears:
```
Assets/HoloToolkit-Examples/UX/Scripts/OptimizeSceneforDeviceType.cs(8,19): error CS0234: The type or namespace name 'XR' does not exist in the namespace 'UnityEngine'. Are you missing an assembly reference?
```

## Steps to reproduce
Open the MixedRealityToolkit-Unity in Unity right after cloning the repo.

## Unity Editor Version
2017.1.x
5.6x

## Mixed Reality Toolkit Release Version
Latest master https://github.com/Microsoft/MixedRealityToolkit-Unity/commit/29ff159d6dee3a6ca6343c7fc305b26a51030488

**Comment by samharper94** [comment]
I'm using the latest master on 2017.1.2 as well and it's working on my end, have you enabled VR support under player settings?

That example scene is for immersive and holo headsets so I imagine it needs 2017.2 to work, have you tried importing without the examples?

**Comment by StephenHodgson** [comment]
I had originally fixed this in #1309 but that PR was replaced by #1339

**Comment by StephenHodgson** [comment]
Duplicate of #1304
